server/src/test/java/org/elasticsearch/cluster/routing/allocation/decider/DiskThresholdDeciderUnitTests.java

```diff
@@ -342,14 +342,19 @@ public void testShardSizeAndRelocatingSize() {
         node = new RoutingNode("node1", new DiscoveryNode("node1", buildNewFakeTransportAddress(), emptyMap(), emptySet(),
             Version.CURRENT), test_0, test_1.getTargetRelocatingShard(), test_2, other_0.getTargetRelocatingShard());
         if (other_0.primary()) {
-            assertEquals(10100L, DiskThresholdDecider.sizeOfRelocatingShards(node, allocation, false, "/dev/null"));
-            assertEquals(10090L, DiskThresholdDecider.sizeOfRelocatingShards(node, allocation, true, "/dev/null"));
+            assertEquals(10100L, sizeOfRelocatingShards(allocation, node, false, "/dev/null"));
+            assertEquals(10090L, sizeOfRelocatingShards(allocation, node, true, "/dev/null"));
         } else {
-            assertEquals(100L, DiskThresholdDecider.sizeOfRelocatingShards(node, allocation, false, "/dev/null"));
-            assertEquals(90L, DiskThresholdDecider.sizeOfRelocatingShards(node, allocation, true, "/dev/null"));
+            assertEquals(100L, sizeOfRelocatingShards(allocation, node, false, "/dev/null"));
+            assertEquals(90L, sizeOfRelocatingShards(allocation, node, true, "/dev/null"));
         }
     }
 
+    public long sizeOfRelocatingShards(RoutingAllocation allocation, RoutingNode node, boolean subtractShardsMovingAway, String dataPath) {
```

Comment:
nit: I would prefer this to have same parameter order as the underlying method, i.e., move allocation last.

Reply:
Thanks, yes, see 895ffa8